addons/account_accountant/account_accountant_data.xml

```diff
@@ -8,7 +8,7 @@
             web_icon="account,static/description/icon.png"
             id="account.menu_finance"
             sequence="40"/>
-        <menuitem id="account.menu_account_config" name="Settings" parent="account.menu_finance_configuration" sequence="0" groups="base.group_configuration"/>
+        <menuitem id="account.menu_account_config" name="Settings" parent="account.menu_finance_configuration" sequence="0"/>
```

Comment:
Are you based on the group no one of the parent ? Or should it be displayed everytime ?